core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/remote/ListStatsCommand.java

```diff
@@ -148,4 +138,19 @@ public void setParameters(
 		}
 	}
 
+	@Override
+	public Void computeResults(
```

Comment:
this seems to be a problem - for liststats to work properly for the REST endpoint, it needs to return a string or list of strings I think (similar to listindex or listadapter)